docker/1.6.0/py2/Dockerfile.cpu

```diff
@@ -56,8 +86,17 @@ RUN pip install --no-cache --upgrade \
     scipy==1.2.2 \
     urllib3==1.24 \
     python-dateutil==2.8.0 \
+    mpi4py==3.0.2 \
+    gluoncv==0.6.0 \
```

Comment:
do we need to have tests for gluon cv if we're adding anyway for mxnet horovod?

Reply:
The `gluon-cv` package was required because horovod+MXNet test uses some of the gluon-cv API in the EC2 and EKS test.